From ac9a8d61b3921493d90f57c60fccdd48202e426d Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Tue, 28 Oct 2025 10:35:12 +0100
Subject: [PATCH] arm64: dts: st: remove #access-controller-cells for RISABs on
 stm32mp251x

There is no need to declare the RISAB nodes as access-controllers,
simply remove the #access-controller-cells property.

Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Change-Id: Ia9b0d8b44020f91b017e636ea7b71b09c2737c51
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/503471
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Tested-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Domain-Review: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Reviewed-by: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
---
 arch/arm64/boot/dts/st/stm32mp251.dtsi | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/arch/arm64/boot/dts/st/stm32mp251.dtsi b/arch/arm64/boot/dts/st/stm32mp251.dtsi
index 3dea9a920bc5..f51dab8c8f15 100644
--- a/arch/arm64/boot/dts/st/stm32mp251.dtsi
+++ b/arch/arm64/boot/dts/st/stm32mp251.dtsi
@@ -2459,7 +2459,6 @@ risab1: risab@420f0000 {
 			reg = <0x420f0000 0x1000>;
 			clocks = <&scmi_clk CK_SCMI_ICN_LS_MCU>;
 			st,mem-map = <0xa000000 0x20000>;
-			#access-controller-cells = <1>;
 		};
 
 		risab2: risab@42100000 {
@@ -2467,7 +2466,6 @@ risab2: risab@42100000 {
 			reg = <0x42100000 0x1000>;
 			clocks = <&scmi_clk CK_SCMI_ICN_LS_MCU>;
 			st,mem-map = <0xa020000 0x20000>;
-			#access-controller-cells = <1>;
 		};
 
 		risab3: risab@42110000 {
@@ -2475,7 +2473,6 @@ risab3: risab@42110000 {
 			reg = <0x42110000 0x1000>;
 			clocks = <&scmi_clk CK_SCMI_ICN_LS_MCU>;
 			st,mem-map = <0xa040000 0x20000>;
-			#access-controller-cells = <1>;
 		};
 
 		risab4: risab@42120000 {
@@ -2483,7 +2480,6 @@ risab4: risab@42120000 {
 			reg = <0x42120000 0x1000>;
 			clocks = <&scmi_clk CK_SCMI_ICN_LS_MCU>;
 			st,mem-map = <0xa060000 0x20000>;
-			#access-controller-cells = <1>;
 		};
 
 		risab5: risab@42130000 {
@@ -2491,7 +2487,6 @@ risab5: risab@42130000 {
 			reg = <0x42130000 0x1000>;
 			clocks = <&scmi_clk CK_SCMI_ICN_LS_MCU>;
 			st,mem-map = <0xa080000 0x20000>;
-			#access-controller-cells = <1>;
 		};
 
 		risab6: risab@42140000 {
@@ -2499,7 +2494,6 @@ risab6: risab@42140000 {
 			reg = <0x42140000 0x1000>;
 			clocks = <&scmi_clk CK_SCMI_ICN_LS_MCU>;
 			st,mem-map = <0xa0a0000 0x20000>;
-			#access-controller-cells = <1>;
 			status = "disabled";
 		};
 
-- 
2.53.0

